### PR TITLE
GPV-902 Change redirection syntax throughout

### PR DESCRIPTION
**Pull request by mnemesh-pivotal**
- Cleaned up the awkward redirection syntax in the code and README.md
- Changed the way the activity is monitored on multi-user to prevent a
  possible early exit
- Changed TABS to spaces that were missed in prior commits
- Added {} to some variables that were missed in prior commits

Co-authored-by: Mike Nemesh <nemeshm@vmware.com>
Co-authored-by: Brandon Tat <tatb@vmware.com>

### CODE REVIEW

**Review by Chibin — APPROVED** [review]

File: 02_init/rollout.sh [hunk 1]
```diff
@@ -18,28 +18,28 @@ function set_segment_bashrc()
   echo "  . /etc/bashrc" >> ${PWD}/segment_bashrc
   echo "fi" >> ${PWD}/segment_bashrc
   echo "source /usr/local/greenplum-db/greenplum_path.sh" >> ${PWD}/segment_bashrc
-  echo "export LD_PRELOAD=/lib64/libz.so.1 ps" >> ${PWD}/segment_bashrc
+  echo "export LD_PRELOAD=${LD_PREOAD}" >> ${PWD}/segment_bashrc
```
[the remark this answers is not in the document]
Comment (degaurab) [reply 02_init/rollout.sh:21]: Fixed in https://github.com/pivotal/TPC-DS/pull/36/files

File: 02_init/rollout.sh [hunk 2]
```diff
@@ -18,28 +18,28 @@ function set_segment_bashrc()
...
-        scp ${PWD}/segment_bashrc ${ext_host}:${ADMIN_HOME}/.bashrc
+    if [ "${MASTER_HOST}" != "${shortname}" ]; then
+      bashrc_exists=$(ssh ${ext_host} "find ~ -name .bashrc | grep -c .")
+      if [ ${bashrc_exists} -eq 0 ]; then
+        echo "copy new .bashrc to ${ext_host}:~${ADMIN_USER}"
+        scp ${PWD}/segment_bashrc ${ext_host}:~${ADMIN_USER}/.bashrc
       else
-        count=$(ssh ${ext_host} "grep greenplum_path ~/.bashrc" 2> /dev/null | wc -l)
-        if [ "$count" -eq "0" ]; then
+        count=$(ssh ${ext_host} "grep -c greenplum_path ~/.bashrc")
```
[the remark this answers is not in the document]
Comment (degaurab) [reply 02_init/rollout.sh:34]: Fixed in https://github.com/pivotal/TPC-DS/pull/36/files

File: 02_init/rollout.sh [hunk 3]
```diff
@@ -18,28 +18,28 @@ function set_segment_bashrc()
...
+        if [ ${count} -eq 0 ]; then
           echo "Adding greenplum_path to ${ext_host} .bashrc"
           ssh ${ext_host} "echo \"source ${GREENPLUM_PATH}\" >> ~/.bashrc"
         fi
-        count=$(ssh ${ext_host} "grep LD_PRELOAD ~/.bashrc" 2> /dev/null | wc -l)
-        if [ "$count" -eq "0" ]; then
+        count=$(ssh ${ext_host} "grep -c LD_PRELOAD ~/.bashrc")
```
[the remark this answers is not in the document]
Comment (degaurab) [reply 02_init/rollout.sh:39]: Fixed in https://github.com/pivotal/TPC-DS/pull/36/files

File: 07_multi_user/rollout.sh [hunk 4]
```diff
@@ -5,95 +5,95 @@ set -e
...
-	exit 0
+  echo "MULTI_USER_COUNT set at 0 so exiting..."
+  exit 0
 fi
 
-function get_psql_count()
+function get_running_jobs_count()
 {
-	psql_count=$(ps -ef | grep psql | grep multi_user | grep -v grep | wc -l)
+  job_count=$(ps -fu "${ADMIN_USER}" | grep -c "TPC-DS/07_multi_user/test.sh")
```
[the remark this answers is not in the document]
Comment (degaurab) [reply 07_multi_user/rollout.sh:14]: Fixed in https://github.com/pivotal/TPC-DS/pull/36/files

File: 07_multi_user/rollout.sh [hunk 5]
```diff
@@ -5,95 +5,95 @@ set -e
...
+  echo "${job_count}"
 }
 
 function get_file_count()
 {
-	file_count=$(ls ${TPC_DS_DIR}/log/end_testing* 2> /dev/null | wc -l)
+  file_count=$(find ${TPC_DS_DIR}/log -maxdepth 1 -name 'end_testing*' | grep -c .)
```
[the remark this answers is not in the document]
Comment (degaurab) [reply 07_multi_user/rollout.sh:20]: Fixed in https://github.com/pivotal/TPC-DS/pull/36/files

**Pull request merged**